src/compiler/scala/tools/nsc/ast/parser/Scanners.scala

```diff
@@ -854,7 +854,12 @@ trait Scanners extends ScannersCommon {
       } else unclosedStringLit()
     }
 
-    private def unclosedStringLit(): Unit = syntaxError("unclosed string literal")
+    private def unclosedStringLit(seenEscapedQuoteInInterpolation: Boolean = false): Unit = {
+      val note =
+        if (seenEscapedQuoteInInterpolation) "; note that `\\\"` no longer closes single-quoted interpolated string literals since 2.13.6, you can use a triple-quoted string instead"
```

Comment:
This suggests to move from `s"..."` to `s"""..."""`. The two behave the same, because escapes are handled by `s` in both cases.

To me the main goal of this PR is to avoid the surprise of going from `"she: \"me?\""` to `s"she: \"me?\""`. The discrepancy between `"""yes\no"""` and `s"""yes\no"""` is much less sublte and should maybe also be addressed somehow...